### PR TITLE
[16.0][MIG] currency_rate_update: Migration to 16.0

### PR DESCRIPTION
**Pull request by AEstLo**

### CODE REVIEW

**Review by AungKoKoLin1997 — COMMENTED** [review]
Functional review and tested in local environment. It works as expected.
Screenshot from 2022-11-30 11-57-53

**Comment by alexis-via** [comment]
I made a PR to improve this PR: https://github.com/AEstLo/currency/pull/1
Once my improvement PR will be merged, I will approve this PR (I already tested it and it works well).

**Comment by AEstLo** [comment]
Thanks @alexis-via 
I already reviewed, approved and merged your PR. The pre-commit fails because of this
`fatal: could not read Username for 'https://gitlab.com/': No such device or address`

**Comment by AungKoKoLin1997** [comment]
@AEstLo Please rebase your PR. pre-commit is already fixed by this [PR](https://github.com/OCA/currency/pull/144).

**Review by alexis-via — APPROVED** [review]

File: currency_rate_update/models/res_currency_rate_provider_ECB.py
Comment (victoralmau) [inline currency_rate_update/models/res_currency_rate_provider_ECB.py]: According to https://www.ecb.europa.eu/services/contacts/working-hours/html/index.en.html the correct time is 17:30, and in UTC+0 it would be 18:30, therefore 19 should be kept (as it was).

**Review by victoralmau — COMMENTED** [review]
Please, cherry-pick https://github.com/OCA/currency/pull/145 to the commit history

Can you unify all your changes in the migration commit?

File: currency_rate_update/models/res_currency_rate_provider_ECB.py
Comment (alexis-via) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: @victoralmau No, you are wrong. Please read the comment, it has the correct URL

Comment (alexis-via) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: Correct URL is : https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html
It is written in the comment of the code, so please read the comment !

Comment (alexis-via) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]:
```
The reference rates are usually updated around 16:00 CET on every working day
```

**Comment by alexis-via** [comment]
a rebase should fix the pre-commit error.

File: currency_rate_update/models/res_currency_rate_provider_ECB.py
Comment (victoralmau) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: I have read the message, but on the web an hour appears on one side and a different one on the other side. 

Anyway, in a few days I'll create a PR to try remove it and improve it with `next_run` and `last_sucessfull_run`.

Comment (alexis-via) [reply currency_rate_update/models/res_currency_rate_provider_ECB.py]: No. For the update of exchange rates, there is just one time: it's 16:00 CET ; it is very clear on the page https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/index.en.html And, from my experience, it is really 16:00 CET in real life. So don't change it.

**Comment by AEstLo** [comment]
@alexis-via @AungKoKoLin1997 Rebased

**Comment by kanda999** [comment]
Funtion review: It workes as expected.

**Comment by alexis-via** [comment]
git rm currency_rate_update/i18n/en.po

Then pre-commit should be green.

**Comment by AEstLo** [comment]
done, now this is green :)
Thanks

**Comment by Gelo-fl** [comment]
Hi @dreispt,

This module is merged, however the translations are locked:
[currency-16.0-currency_rate_update](https://translation.odoo-community.org/projects/currency-16-0/currency-16-0-currency_rate_update/)

Can I know why?

Thanks in advance,

**Comment by Gelo-fl** [comment]
> Hi @dreispt,
> 
> This module is merged, however the translations are locked: [currency-16.0-currency_rate_update](https://translation.odoo-community.org/projects/currency-16-0/currency-16-0-currency_rate_update/)
> 
> Can I know why?
> 
> Thanks in advance,

Please,
Could someone please unblock the translation of this module?

**Pull request merged**
